### PR TITLE
Fix compile failures on windows

### PR DESCRIPTION
**Pull request by tajila**
Fix compile failures on windows

Signed-off-by: tajila <atobia@ca.ibm.com>

### CODE REVIEW

**Comment by DanHeidinga** [comment]
Jenkins test sanity plinux

File: runtime/vm/createramclass.cpp [hunk 1]
```diff
@@ -2804,10 +2804,10 @@ internalCreateRAMClassFromROMClass(J9VMThread *vmThread, J9ClassLoader *classLoa
 					}
 
 					packageUTF = J9UTF8_DATA(J9ROMCLASS_CLASSNAME(packageRomClass));
-					pkgNameLength = packageNameLength(packageRomClass);
+					pkgNameLength = (U_32) packageNameLength(packageRomClass);
```
Comment (keithc-ca) [inline runtime/vm/createramclass.cpp]: Changing the declared type to `UDATA` would remove the need for these two changes.

Comment (DanHeidinga) [reply runtime/vm/createramclass.cpp]: I started down the same route but `findModuleForPackage` takes a U_32, and then truncates it to a U_16.  Fixing that is outside the scope of unblocking the windows builds.

Comment (keithc-ca) [reply runtime/vm/createramclass.cpp]: I was only suggesting the one-line change based on the other uses of `packageNameLength` I could find, but I agree this is close enough.

**Pull request merged**